src/main/java/dev/j3fftw/litexpansion/Items.java

```diff
@@ -272,6 +272,46 @@
         LoreBuilderDynamic.powerPerTick(RubberSynthesizer.ENERGY_CONSUMPTION)
     );
     //// Solar panels
+    public static final SlimefunItemStack BASIC_SOLAR_PANEL = new SlimefunItemStack(
+        "BASIC_SOLAR_PANEL",
+        Material.BLUE_GLAZED_TERRACOTTA,
+        "&3&lBasic Solar Panel",
+        "&9Works at Night",
+        "",
+        LoreBuilderDynamic.powerBuffer(AdvancedSolarPanel.BASIC_STORAGE),
+        LoreBuilderDynamic.powerPerTick(AdvancedSolarPanel.BASIC_DAY_RATE) + " (Day)",
+        LoreBuilderDynamic.powerPerTick(AdvancedSolarPanel.BASIC_NIGHT_RATE) + " (Night)"
+    );
+    public static final SlimefunItemStack LOW_VOLTAGE_SOLAR_PANEL = new SlimefunItemStack(
+        "LOW_VOLTAGE_SOLAR_PANEL",
+        Material.RED_GLAZED_TERRACOTTA,
+        "&c&lLow Voltage Solar Panel",
+        "&9Works at Night",
+        "",
+        LoreBuilderDynamic.powerBuffer(AdvancedSolarPanel.LOW_VOLTAGE_STORAGE),
+        LoreBuilderDynamic.powerPerTick(AdvancedSolarPanel.LOW_VOLTAGE_DAY_RATE) + " (Day)",
+        LoreBuilderDynamic.powerPerTick(AdvancedSolarPanel.LOW_VOLTAGE_NIGHT_RATE) + " (Night)"
+    );
+    public static final SlimefunItemStack MEDIUM_VOLTAGE_SOLAR_PANEL = new SlimefunItemStack(
+        "MEDIUM_VOLTAGE_SOLAR_PANEL",
+        Material.ORANGE_GLAZED_TERRACOTTA,
+        "&6&lMedium Voltage Solar Panel",
+        "&9Works at Night",
+        "",
+        LoreBuilderDynamic.powerBuffer(AdvancedSolarPanel.MEDIUM_VOLTAGE_STORAGE),
+        LoreBuilderDynamic.powerPerTick(AdvancedSolarPanel.MEDIUM_VOLTAGE_DAY_RATE) + " (Day)",
+        LoreBuilderDynamic.powerPerTick(AdvancedSolarPanel.MEDIUM_VOLTAGE_NIGHT_RATE) + " (Night)"
+    );
+    public static final SlimefunItemStack HIGH_VOLTAGE_SOLAR_PANEL = new SlimefunItemStack(
+        "HIGH_VOLTAGE_SOLAR_PANEL",
+        Material.YELLOW_GLAZED_TERRACOTTA,
+        "&e&lHigh Voltage Solar Panel",
+        "&9Works at Night",
+        "",
+        LoreBuilderDynamic.powerBuffer(AdvancedSolarPanel.HIGH_VOLTAGE_STORAGE),
+        LoreBuilderDynamic.powerPerTick(AdvancedSolarPanel.HIGH_VOLTAGE_DAY_RATE) + " (Day)",
+        LoreBuilderDynamic.powerPerTick(AdvancedSolarPanel.HIGH_VOLTAGE_NIGHT_RATE) + " (Night)"
+    );
     public static final SlimefunItemStack ADVANCED_SOLAR_PANEL = new SlimefunItemStack(
```

Comment:
New ones shouldnt work at night

Reply:
Then they wont work at all cuz of the only nightime rate bug

Reply:
Then that issue should be fixed. I still dont want these to work at night. No matter the issue